.buildkite/pipeline.yml

```diff
@@ -3,7 +3,6 @@
 env:
   SETUP_GVM_VERSION: 'v0.5.0'
   DOCKER_COMPOSE_VERSION: '1.25.5'
-  DOCKER_REGISTRY: "docker.elastic.co"
   DOCKER_IMAGE: "${DOCKER_REGISTRY}/observability-ci/temp-fleet-server" # needs to rename after tests
   DOCKER_IMAGE_TAG: ""
```

Comment:
nit. Maybe this could be removed, since it is created and used inside the script.

Reply:
ok